src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -5,270 +5,279 @@
       attach="#reg-add-edit-org-person"
     />
 
-    <v-expand-transition>
-      <ul class="add-person pa-0">
-        <li class="add-person-container px-6 py-10">
-          <div class="meta-container">
-
-            <!-- FUTURE: move header text to resource file so this component is generic -->
-            <!-- Title for org -->
-            <label class="add-org-header" v-if="isOrg">
-              <span v-if="activeIndex === -1">Add Business or Corporation</span>
-              <span v-else>Edit Business or Corporation</span>
-            </label>
-
-            <!-- Title for person -->
-            <label class="add-org-header" v-if="isPerson">
-              <span v-if="activeIndex === -1">Add Person</span>
-              <span v-else>Edit Person</span>
-            </label>
-
-            <!-- The form -->
-            <div class="meta-container__inner">
-              <v-card outlined class="message-box rounded-0" v-if="isCompletingParty && !isRoleStaff">
-                <p>
-                  <strong>Important:</strong> The Completing Party information below is based on your
-                  BC Registries account information. Your name and email cannot be changed here. Name
-                  and email changes must be made through your account settings.
-                </p>
-                <p>
-                  If you make changes to your address below, please update your address in the account
-                  settings after you have completed this filing to ensure your information is up to date.
-                </p>
-              </v-card>
-
-              <v-form
-                ref="addPersonOrgForm"
-                class="appoint-form"
-                :class="{ 'mt-8': isCompletingParty && !isRoleStaff}"
-                v-model="addPersonOrgFormValid"
-                v-on:submit.prevent
+    <section class="px-6 py-10" :class="{ 'invalid-section': !addPersonOrgFormValid }">
```

Comment:
Please hide whitespace (⚙️ icon at top of Files Changed page) to see what actually changed.